images/bugzilla-backporter/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM centos:7
+LABEL maintainer="skuznets@redhat.com"
+
+ADD bugzilla-backporter /usr/bin/bugzilla-backporter
+RUN yum -y install backporter && yum clean all && rm -rf /var/cache/yum
```

Comment:
I am trying to install the package which the tool depends on, not sure its necessary though - since I am not installing any of the other dependencies.

Reply:
@GeoBK Can you elaborate more in why the tool needs this `backporter` package?

Reply:
I am not really sure its needed, I was mimicing the format in another file which installed graphviz but its only now that I realize that it was probably an external package it was installing.

Reply:
@droslean Does it look ok now?